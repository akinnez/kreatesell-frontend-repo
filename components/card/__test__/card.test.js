import renderer from "react-test-renderer";
import { Card } from "../index";

const MockComponent = () => <div>card mock Children</div>;
it("renders a snapshot of the Card component", () => {
  const card = renderer
    .create(
      <Card>
        <MockComponent />
      </Card>
    )
    .toJSON();
  expect(card).toMatchSnapshot();
});
