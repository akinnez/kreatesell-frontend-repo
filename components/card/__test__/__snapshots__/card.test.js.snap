// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of the Card component 1`] = `
<div
  className="jsx-3923066720 card"
>
  <div>
    card mock Children
  </div>
</div>
`;
